File: results/py/DIFFERENCE_BETWEEN_HIGHEST_AND_LEAST_FREQUENCIES_IN_AN_ARRAY_1.cpp
int f_filled ( int arr [ ] , int n ) {
  std :: map < int , int > mp ;
  mp [ arr [ i ] ] ++ ;
  int maxCount = 0 ;
  ;
  int minCount = n ;
  for ( std :: map < int , int > :: iterator it = mp . begin ( ) ;
  it != mp . end ( ) ;
  it ++ ) {
    maxCount = std :: max ( maxCount , it -> second ) ;
    minCount = std :: min ( minCount , it -> second ) ;
  }
  return maxCount - minCount ;
}

File: results/java_codex/DIFFERENCE_BETWEEN_HIGHEST_AND_LEAST_FREQUENCIES_IN_AN_ARRAY_1.java
int findDiff(int arr[], int n) {
        unordered_map<int, int> mp;
        for (int i = 0; i < n; i++) {
            if (mp.find(arr[i]) != mp.end()) {
                mp[arr[i]]++;
            } else {
                mp[arr[i]] = 1;
            }
        }
        int max_count = 0, min_count = n;
        for (auto x : mp) {
            max_count = max(max_count, x.second);
            min_count = min(min_count, x.second);
        }
        return (max_count - min_count);
    }